packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -426,7 +429,19 @@ export class LitDevTutorial extends LitElement {
           html`${`Could not fetch step description. Invalid URL /${
             this._projectLocation
           }/#${this._idxToSlug(this._idx)}`}`,
-        complete: (response) => html`${unsafeHTML(response)}`,
+        complete: (response) => {
+          // Must use DOM parser to attach declarative shadow roots
+          const domResponse = (domParser.parseFromString as any)(
```

Comment:
why is the `as any` needed? Something wrong with the `parseFromString` signature?

Reply:
the options argument is not in TS yet. I assume because it's only Chrome for now

Reply:
Ah ok. Could you add a comment?

Reply:
aye aye